package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "html-pdf-converter",
+  "description": "An HTML to PDF conversion service wrapper",
+  "version": "1.0.0",
+  "license": "MIT",
+  "author": "Joe Chapman joseph@creatify.io",
+  "repository": "git+https://github.com/UKHomeOffice/html-pdf-converter.git",
+  "bugs": {
+    "url": "https://github.com/UKHomeOffice/html-pdf-converter/issues"
+  },
+  "homepage": "https://github.com/UKHomeOffice/html-pdf-converter#readme",
+  "main": "index.js",
+  "scripts": {
+    "start": "node .",
+    "test": "mocha",
+    "lint": "eslint .",
+    "ci": "npm test && npm lint"
```

Comment:
I have a strong preference for `npm test` running all of the tests for discoverability reasons. I'd move mocha to `unit` or `test:unit` (and correspondingly have `test:lint`) and then have `npm test` run all of the tests.